explorer/client/src/components/transactions-for-id/TxForID.tsx

```diff
@@ -37,76 +37,82 @@ type TxnData = {
-                                    styles[x.status.toLowerCase()]
-                                )}
+            {showData.map((x, index) => (
+                <div key={`txid-${index}`} className={styles.txrow}>
+                    <div className={styles.txid}>
+                        <Longtext
+                            text={x.txId}
+                            category="transactions"
+                            isLink={true}
+                        />
+                    </div>
+                    <div className={styles.txtype}>{x.kind}</div>
+                    <div
+                        className={cl(
+                            styles.txstatus,
+                            styles[x.status.toLowerCase()]
+                        )}
+                    >
+                        {x.status === 'success' ? '\u2714' : '\u2716'}
+                    </div>
+                    <div className={styles.txadd}>
+                        <div>
+                            From:
+                            <Link
+                                className={styles.txlink}
+                                to={'addresses/' + x.From}
                             >
-                                {x.status === 'success' ? '\u2714' : '\u2716'}
-                            </div>
-                            <div className={styles.txadd}>
-                                <div>
-                                    From:
-                                    <Link
-                                        className={styles.txlink}
-                                        to={'addresses/' + x.From}
-                                    >
-                                        {truncate(x.From, 14, '...')}
-                                    </Link>
-                                </div>
-                                {x.To && (
-                                    <div>
-                                        To :
-                                        <Link
-                                            className={styles.txlink}
-                                            to={'addresses/' + x.To}
-                                        >
-                                            {truncate(x.To, 14, '...')}
-                                        </Link>
-                                    </div>
-                                )}
-                            </div>
+                                {truncate(x.From, 14, '...')}
+                            </Link>
                         </div>
-                    ))}
+                        {x.To && (
+                            <div>
+                                To :
+                                <Link
+                                    className={styles.txlink}
+                                    to={'addresses/' + x.To}
+                                >
+                                    {truncate(x.To, 14, '...')}
```

Comment:
nit: Could you follow up with a PR to define a constant for `14`?

Reply:
Yes - will do